src/libstd/sys/unix/backtrace/printing/mod.rs

```diff
@@ -15,5 +15,5 @@ pub use self::imp::*;
 mod imp;
 
 #[cfg(not(any(target_os = "macos", target_os = "ios")))]
-#[path = "libbacktrace.rs"]
+#[path = "../../../gnu/libbacktrace.rs"]
```

Comment:
Can this file instead live inside of `src/libstd/sys/common`? It can be `#[cfg]`'d off there and just `pub use`'d here